config/cesm/machines/config_compilers.xml

```diff
@@ -511,6 +503,38 @@ using a fortran linker.
   <TRILINOS_PATH MPILIB="mpich2">$ENV{TRILINOS_PATH}</TRILINOS_PATH>
 </compiler>
 
+<compiler MACH="athena">
```

Comment:
These additions are suspect.

Reply:
seem to be a repeat of lines starting at 474

Reply:
OK, i will remove